packages/fluent-theme/src/fluent/styles/Dropdown.styles.ts

```diff
@@ -66,11 +65,16 @@ export const DropdownStyles = (props: IDropdownStyleProps) => {
         selectors: {
           // Title and border states. For :hover and :focus even if the styles are the same we need to keep them separate for specificity
           // reasons in order :active borderColor to work.
-          ['&:hover .ms-Dropdown-title']: { color: semanticColors.menuItemTextHovered, borderColor: palette.neutralPrimary },
-          ['&:focus .ms-Dropdown-title']: { color: semanticColors.menuItemTextHovered, borderColor: palette.neutralPrimary },
+          ['&:hover .ms-Dropdown-title']: {
+            color: semanticColors.menuItemTextHovered,
+            borderColor: !isOpen ? palette.neutralPrimary : CommunicationColors.primary
```

Comment:
Should we be using `palette.themePrimary` or `CommunicationColors.primary` for the theme? I'm leaning toward using `palette` since that will be easier to merge into master at some point. I hadn't thought about this and have been using `CommunicationColors` in my changes. Whichever way we go, we should be consistent.

Reply:
Discussed this and arrived at guidance for this. We'll use the `theme.palette` object wherever possible, referring to Fluent palettes like `NeutralColors` only if we need to use a color that isn't in the theme yet.